app/controllers/api/connect/v3/systems/systems_controller.rb

```diff
@@ -3,6 +3,17 @@ class Api::Connect::V3::Systems::SystemsController < Api::Connect::BaseControlle
   before_action :authenticate_system
 
   def update
+    if params[:online_at].present?
+      params[:online_at].each do |online_at|
+        dthours = online_at.split(':')
+        @system_uptime = SystemUptime.create!(system_id: @system.id, online_at_day: dthours[0], online_at_hours: dthours[1])
+        logger.info(N_("Added uptime information for system '%s'") % @system.id)
+      rescue ActiveRecord::RecordNotUnique
+        logger.info(N_("Uptime information existing for system '%s'") % @system.id)
+      end
+      SystemUptime.where("online_at_day < '#{90.days.ago.to_date}'").delete_all
```

Comment:
I'm wondering if this has impact on the RMT performance if it is called on every request. Did you look into this?

Reply:
Let's discuss what the options are on wed morning meeting